hw/ip/aes/dv/aes_sim_cfg.hjson

```diff
@@ -67,6 +67,11 @@
       uvm_test: aes_stress_test
       uvm_test_seq: aes_stress_vseq
     }
+    {
+      name: aes_b2b
+      uvm_test: aes_b2b_test
+      uvm_test_seq: aes_stress_vseq
```

Comment:
Is this intended that `aes_b2b` test still uses `aes_stress_vseq`? Do you mean `aes_b2b_vseq`?

Reply:
Hi @cindychip this intentional.
all or almost all of my tests will use the same sequence.
the only difference is the configuration which is done in the test.

This is another approach to what is generally used in OpenTitan.
Eventually, I will split up my env_cfg and my seq_cfg. 
We agreed a while back that this approach is accepted.

Reply:
@rasmus-madsen do you have a plan to enable line #36 in the `aes_sim_cfg.dv`? Are the `aes_stress_vseq `and `aes_stress_all_vseq ` (enable by line #32) different? I check the description of the aes_stress_vseq and it seems to me that is similar to the goals of the aes_stress_all_vseq. Maybe, I am missing some points but it is worth to check.

Reply:
@tunghoang290780 
conceptually they are very similar.
the stress_all_vseq stems from the autogenerated part of the testbench and is used for chaining sequences.
and with this approach there is just one testcase - the base. which is then morphed into whatever, based on the sequence referenced.

my approach is sort of the other way around. 
there is just one "top" sequence that is then configured from a testcase.
I think this is closer to what was originally intended by UVM.

there are some upsides to both methods.

Aes currently serves as an example of this other method. so we can weigh these pros and cons.

We can talk about these offline if you have an interest in the details

Reply:
@rasmus-madsen ok, we can talk offline. I think this might be out of the scope of this PR